source/includes/harvest/_job_openings.md

```diff
@@ -341,11 +344,25 @@ job_id | The ID of the job on which to add new openings.
 
 Parameter | Required | Type | Description
 --------- | ----------- | ----------- | -----------
-openings | yes | Array | This an array of opening IDs, which contain one element.
+openings | yes | Array | This an array of hashes, which can contain an opening ID and custom fields.
 openings.opening_id | yes | Array | This is a string that contains an opening_id. One new opening will be created for each hash element in the array. Opening ID is not required to have a string value and may be null.  Greenhouse has an internal limit of 100 open openings. If you attempt to create more than 100 openings in a single request, or if this request would create more than 100 open openings, the request will fail.
+openings.custom_fields[] | No | Array | Array of hashes containing new custom field values. Passing an empty array does nothing. If you have any required custom fields configured for openings, they must be supplied for each new opening, or the request will fail.
```

Comment:
I think we should add a link to the `Custom Field Parameters` section below so that it's clear where to look for more info on custom field parameters.